src/utils/mdast-to-vdom.js

```diff
@@ -61,6 +62,7 @@ class VDOMTransformer {
         return true;
       });
     this._handlers.image = image(options);
+    this._handlers.embed = link(options);
```

Comment:
Typo: should be `this._handlers.link =…`

Reply:
Btw. I'm unclear why your tests are passing, given that you register the wrong handler.

Reply:
Fixed in https://github.com/adobe/hypermedia-pipeline/pull/120/commits/6386202b2e004517b064f2ed2991b8dded8739e9